test/JDBC/expected/BABEL-937-vu-verify.out

```diff
@@ -0,0 +1,136 @@
+select * from babel_937_not_append_kp
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#nvarchar
+{"name": "Mike", "skills": ["C#", "SQL"]}#!#{"name": "Mike", "skills": ["C#", "SQL"]}#!#{"name": "Mike", "surname": "Smith"}#!#{"name": null, "skills": ["C#", "SQL"], "surname": "Smith"}
+~~END~~
+
+
+select * from babel_937_not_append_knp_1
+go
+~~START~~
+nvarchar#!#nvarchar
+{"name": "Mike", "skills": ["C#", "SQL"], "surname": "Smith"}#!#{"name": "Mike", "skills": ["C#", "SQL"], "surname": "Smith"}
+~~END~~
+
+select * from babel_937_not_append_knp_2
+go
+~~START~~
+nvarchar
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: property cannot be found on the specified JSON path)~~
+
+select * from babel_937_not_append_knp_3
+go
+~~START~~
+nvarchar
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: property cannot be found on the specified JSON path)~~
+
+
+select * from babel_937_append_kp_notarr_1
+go
+~~START~~
+nvarchar#!#nvarchar
+{"name": "John", "skills": ["C#", "SQL"]}#!#{"name": "John", "skills": ["C#", "SQL"]}
+~~END~~
+
+select * from babel_937_append_kp_notarr_2
+go
+~~START~~
+nvarchar
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: array cannot be found in the specified JSON path)~~
+
+select * from babel_937_append_kp_notarr_3
+go
+~~START~~
+nvarchar
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: array cannot be found in the specified JSON path)~~
+
+
+select * from babel_937_append_kp_arr
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#nvarchar
+{"name": "John", "skills": ["C#", "SQL", "Azure"]}#!#{"name": "John", "skills": ["C#", "SQL", "Azure"]}#!#{"name": "Mike", "skills": ["C#", "SQL", null], "surname": "Smith"}#!#{"name": "Mike", "skills": ["C#", "SQL", null], "surname": "Smith"}
+~~END~~
+
+
+select * from babel_937_append_knp_notarr_1
+go
+~~START~~
+nvarchar#!#nvarchar
+{"name": "Mike", "skills": ["C#", "SQL"], "surname": ["Smith"]}#!#{"k": [null], "name": "Mike", "skills": ["C#", "SQL"], "surname": "Smith"}
```

Comment:
`surname` is added at the end, but `k` is added at the beginning. Is this expected?

Reply:
Yes, This is an expected and known behavior when using postgres functions are used.

Reply:
In that case, please add some comments in the test file that the result can be different between Babelfish and T-SQL.

Reply:
Resolved in latest commit.